tcms/core/ajax.py

```diff
@@ -194,105 +191,79 @@ class Objects(object):
         __all__ = ['plan', 'case', 'run']
 
         def __init__(self, request, template_name):
-            self.request = request
             self.template_name = template_name
-            for o in self.__all__:
-                if request.GET.get(o):
-                    self.object = o
-                    self.object_pks = request.GET.getlist(o)
-                    break
+            self._get_object_from_request(request)
+
+        def _get_object_from_request(self, request):
```

Comment:
Breking out into a new method is not needed here b/c this method is only used in the constructor and doesn't add more benefit.